packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -14,7 +14,7 @@ export const getStyles = (props: ITooltipStyleProps): ITooltipStyles => {
       theme.fonts.medium,
       AnimationClassNames.fadeIn200,
       {
-        background: semanticColors.menuBackground,
+        background: palette.white,
```

Comment:
rather than moving to palette color here, how about changing the text color menuItemText.

Reply:
@micahgodbolt if we moved, it can be a break change to migrating from 6.

Reply:
menuItemText is based off of palette.neutralPrimary....so updating neutralPrimary would still update menuItemText.

If there's hard problem with this, thats okay. I just wanted to bring it up.

Reply:
Fabric prefers semanticColors when possible. We should change both to semanticColors instead.

Reply:
I'm unclear about the purpose of this change in tooltip for Fabric 7. The description of the PR is vague, and there are no screenshots to illustrate what's going on. 

Is `menuBackground` the wrong color? As others have said, switching to a palette color brings no semantic meaning, so flipping white out to black might not be correct for tooltip backgrounds.

Reply:
@dzearing @kenotron @micahgodbolt 

I got some questions on the theme:

1. what's the relationship between palette and semantic color?

2. If palette is fixed, did you think consumer can modify part of the semantic color without modifying the palette?

from the code and its comment here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/styles/theme.ts#L182

the mapping only existed when semantic color is not provided.

And my guesses are you will allow consumer to define their semantic color based on their understanding of the mapping between palette and semantic color, right?

If this is true, the questions is that if you mixed using palette and semantic color, how can customer define their own semantic color based on their understanding?

We can't analyze every component to make sure every combination of semantic color and palette work well. The only theme we care is the combination in semantic color is good (used by it's meaning).

So i filed this PR to raise the problem. We should not mix-use the two color system.

Why would i use palette?

Because it was used in fabric 6.

Reply:
The use of `palette` in Fabric 6 is a legacy thing, not an example to follow. We should use semantic colors whenever possible moving forward, meaning the tooltip text color should be changed to `semanticColors.menuItemText` (and revert this line).